ui/src/app/pools/views/Pools.js

```diff
@@ -105,51 +118,66 @@ const Pools = () => {
   const resourcePools = useSelector(resourcePoolSelectors.all);
 
   return (
-    <Row>
-      <Col size={12}>
-        <div>
-          {poolsLoading && (
-            <div className="u-align--center">
-              <Loader text="Loading..." />
-            </div>
-          )}
-          {poolsLoaded && (
-            <MainTable
-              className="p-table-expanding--light"
-              defaultSortDirection="ascending"
-              headers={[
-                {
-                  content: "Name",
-                  sortKey: "name",
-                },
-                {
-                  content: "Machines",
-                  sortKey: "machines",
-                },
-                {
-                  content: "Description",
-                  sortKey: "description",
-                },
-                {
-                  content: "Actions",
-                  className: "u-align--right",
-                },
-              ]}
-              expanding={true}
-              paginate={150}
-              rows={generateRows(
-                resourcePools,
-                expandedId,
-                setExpandedId,
-                dispatch,
-                setDeleting
-              )}
-              sortable
-            />
-          )}
-        </div>
-      </Col>
-    </Row>
+    <>
+      {errorMessage ? (
+        <Row>
+          <Col size={12}>
+            <Notification
+              close={() => dispatch(resourcePoolActions.cleanup())}
+              timeout={5000}
+              type="negative"
+            >
+              {errorMessage}
+            </Notification>
+          </Col>
+        </Row>
+      ) : null}
+      <Row>
+        <Col size={12}>
+          <div>
+            {poolsLoading && (
+              <div className="u-align--center">
+                <Loader text="Loading..." />
+              </div>
+            )}
+            {poolsLoaded && (
+              <MainTable
```

Comment:
Why not wrap the conditional around both sections?

Reply:
they're different guards, `poolsLoading` vs `poolsLoaded`.